spring-kafka/src/main/java/org/springframework/kafka/config/MultiMethodKafkaListenerEndpoint.java

```diff
@@ -60,6 +61,43 @@ public MultiMethodKafkaListenerEndpoint(List<Method> methods, @Nullable Method d
 		setBean(bean);
 	}
 
+
+	/**
+	 * Get a method list.
+	 * @return the method list.
+	 * @since 3.2
+	 */
+	public List<Method> getMethods() {
+		return this.methods;
+	}
+
+	/**
+	 * Set a method list.
+	 * @param methods the methods.
+	 * @since 3.2
+	 */
+	public void setMethods(List<Method> methods) {
+		this.methods = methods;
+	}
+
+	/**
+	 * Get a default method.
+	 * @return the default method.
+	 * @since 3.2
+	 */
+	public Method getDefaultMethod() {
+		return this.defaultMethod;
+	}
+
+	/**
+	 * Set a default method.
+	 * @param defaultMethod the default method.
+	 * @since 3.2
+	 */
+	public void setDefaultMethod(Method defaultMethod) {
+		this.defaultMethod = defaultMethod;
+	}
+
```

Comment:
Adding new public methods in a public class. This deserves a brief mention in `whats-new`.